packages/web-components/fast-foundation/src/menu-item/menu-item.ts

```diff
@@ -34,10 +34,10 @@ export class MenuItem extends FASTElement {
      *
      * @public
      * @remarks
-     * HTML Attribute: aria-expanded
+     * HTML Attribute: expanded
      */
-    @attr({ attribute: "aria-expanded", mode: "reflect", converter: booleanConverter })
-    public expanded: boolean = false;
+    @attr({ attribute: "expanded" })
+    public expanded: boolean;
```

Comment:
This needs to be `aria-expanded` or AT will not recognize it as expanded. The attr should stay the same here. I think we you'll want is to keep the boolean attribute converter but change "reflect" to "fromView".

Reply:
We write to the "aria-expanded" attribute in the template, just like we do for disabled:
aria-expanded="${x => x.expanded}"

Reply:
If it's undefined we don't render a value, which fixes the bug

Reply:
gotcha, thanks for the additional context.